doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -1 +1,14 @@
 # Upgrading Pimcore from Version 10.x to Version 11
+
+## IMPORTANT CHANGES TO DO PRIOR THE UPDATE!
+
+### Web2Print
+If you are using Web2Print functionality set the flag "Enable Web2Print documents in default documents view" to true to ensure the PimcoreWebToPrintBundle gets installed by default.
+
+If you are not using Web2Print functionality set the flag to false. 
+> **Caution: This will remove the permissions from all users!**
+
+## CHANGES TO DO AFTER THE UPDATE! 
+
+### Web2Print
+If you do not use Web2Print functionality please make sure to run `bin/console pimcore:documents:cleanup printpage printcontainer` to clean up your system.
```

Comment:
this command should be announced as a new feature in the upgrade notes and have a doc page about the usage.

Recommending that we should start filling this page with some more descriptive content and examples
https://github.com/pimcore/pimcore/edit/11.x/doc/Development_Documentation/22_Administration_of_Pimcore/05_Commandline_Interface.md
or alternatively somewhere under https://pimcore.com/docs/pimcore/11.0/Development_Documentation/Documents/index.html